src/elements.jl

```diff
@@ -188,8 +188,9 @@ coord(v::ElemT) where {ElemT<:Union{Point,AbstractVector}} = v
 coord(v::ElemT) where {ElemT<:Union{HRepElement,SymPoint,Ray,Line}} = v.a
 
 const VRepElementContainer = Union{SymPoint, Ray, Line}
+const FixedVRepElementContainer{N,T} = Union{Ray{N,T}, Line{N,T}, SymPoint{N,T}}
```

Comment:
You can define `const VRepElementContainer{N,T} = Union{SymPoint{N,T}, Ray{N,T}, Line{N,T}}` and remove `FixedVRepElementContainer`